headers/xdp/libxdp.h

```diff
@@ -99,5 +99,6 @@ enum xdp_attach_mode xdp_multiprog__attach_mode(const struct xdp_multiprog *mp);
 struct xdp_program *xdp_multiprog__main_prog(const struct xdp_multiprog *mp);
 struct xdp_program *xdp_multiprog__hw_prog(const struct xdp_multiprog *mp);
 bool xdp_multiprog__is_legacy(const struct xdp_multiprog *mp);
+int xdp_clean_references(int ifindex);
```

Comment:
Let's prefix this with libxdp_ instead of xdp_ and put it at the top with libxdp_get_error() and libxdp_strerror()